config/core/LocalSettings.php

```diff
@@ -452,14 +485,99 @@
 
 
 
+/**
+ *  7) PERMISSIONS
+ *
+ *
+ *
+ **/
+if ( ! isset( $mezaAuthType ) ) {
+	$mezaAuthType = 'anon-edit'; // default: wide open!
+}
+if ( $mezaAuthType === 'anon-edit' ) {
+
+    // allow anonymous read
+    $wgGroupPermissions['*']['read'] = true;
+    $wgGroupPermissions['user']['read'] = true;
+
+    // allow anonymous write
+    $wgGroupPermissions['*']['edit'] = true;
+    $wgGroupPermissions['user']['edit'] = true;
+
+}
+
+else if ( $mezaAuthType === 'anon-read' ) {
+
+    // allow anonymous read
+    $wgGroupPermissions['*']['read'] = true;
+    $wgGroupPermissions['user']['read'] = true;
+
+    // allow anonymous write
```

Comment:
I tagged my comment to the wrong line. Shouldn't this comment (line 515) be "allow only users write" or something meaning the same thing? Line 516 does not allow anyone to edit. Line 517 allows users to edit.